### DISCUSSION TITLE
How to use Fly Check?

### DISCUSSION THREAD

**Discussion by fjchen7** [mrcjkb/rustaceanvim#87] [Q&A] [answered]
I set `checkOnSave = false` in my rust-analyzer configuration and run `:RustLsp flyCheck run` but nothing happened. No LSP diagnostics shown.

How to run `flycheck` command? I want to prevent rust-analyzer from running `cargo check` background automatically and then run it by manual.

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

There's a typo in the `flycheck` LSP request. I'll fix it and push a new release :sweat_smile:

**Reply by mrcjkb** [reply]
Fixed in version 3.9.5.
